static/app/data/forms/organizationGeneralSettings.tsx

```diff
@@ -91,6 +91,16 @@ const formGroups: JsonFormObject[] = [
         label: t('Open Membership'),
```

Comment:
Now that you're here, can you change this to `Open Team Membership`? The title is confusing and makes it look like open org membership.